.github/workflows/publish-npm.yml

```
@@ -72,18 +108,39 @@ jobs:
         working-directory: ${{ inputs.working-directory }}
         run: ${{ inputs.install-command }}
 
```

```suggestion
     - name: Set nightly version from commit
        if: ${{ inputs.tag == 'nightly' }}
        working-directory: ${{ inputs.working-directory }}
        run: |
          CURRENT_VERSION=$(npm pkg get version | tr -d '"')
          NIGHTLY_VERSION="${CURRENT_VERSION}-nightly-${GITHUB_SHA::7}"
          npm pkg set version="$NIGHTLY_VERSION"
```
to enable nightly version